src/panel_gwalker/_gwalker.py

```diff
@@ -93,17 +93,22 @@ class GraphicWalker(ReactComponent):
         doc="""Dark mode preference: 'light', 'dark', 'media'.
         If not provided the appearance is derived from pn.config.theme.""",
     )
-    # This one is added to better explain that currently only 'client' mode is supported
-    # but we envision supporting 'server' mode one day
-    computation: Literal["client"] = param.Selector(
-        objects=["client"],
-        doc="""The computation configuration. Currently only 'client' is supported.""",
+    server_computation: bool = param.Boolean(
+        default=False,
+        doc="""If True the computations will take place on the Panel server or in the Jupyter kernel
+        instead of the client to scale to larger datasets. Default is False.""",
     )
     config: dict = param.Dict(
         doc="""Optional extra Graphic Walker configuration. For example `{"i18nLang": "ja-JP"}`. See the
     [Graphic Walker API](https://github.com/Kanaries/graphic-walker#api) for more details."""
     )
 
+    chart: dict = param.Dict(doc="""The current chart.""")
```

Comment:
I'd probably implement these as asynchronous methods that fetch the chart data.

Reply:
Could you try to do that? I would not know how to do it.

Reply:
Will do. I am continuously torn on this. Having a parameter and an event makes it easy to use the reactive approaches, but it also **feels** wrong and kind of weird. We may have to figure out whether there's some new thing we need to add to Param/Panel to declare methods that you can "trigger". Almost like "Parameterized methods" where the arguments and the triggering event are parameterized.